src/test/java/com/epam/dataservice/PoliceForceServiceTest.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.dataservice;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PoliceForceServiceTest {
+private	static PoliceForceService testService; 
+@BeforeClass
+public static void init(){
+	testService = new PoliceForceService();
+}
+
+@Test
+public void testGetContactNo(){
```

Comment:
Technically there are three tests in this test. 

It should be splited on different test cases with following TDD or BDD. 

Examples: 
should_return_valid_number_for_existed_police_department
should_return_prefix_only_for_unknow_police
should_throw_exception_if_input_is_null
etc. 

You see - one test for one case. In this case, if one of the scenarious fails you will see it clearely in build logs. (in teamcity for example)